doc/docs/master/deploy-manage/deploy/aws-deploy-pachyderm.md

```diff
@@ -7,6 +7,12 @@ For a quick test installation of Pachyderm on AWS (suitable for development), ju
       - Read our [infrastructure recommendations](../ingress/){target=_blank}. You will find instructions on how to set up an ingress controller, a load balancer, or connect an Identity Provider for access control. 
       - If you are planning to install Pachyderm UI. Read our [Console deployment](../console/){target=_blank} instructions. Note that, unless your deployment is `LOCAL` (i.e., on a local machine for development only, for example, on Minikube or Docker Desktop), the deployment of Console requires, at a minimum, the set up on an [Ingress](../ingress/#ingress).
 
+!!! Attention 
+    We are now shipping Pachyderm with an **optional embedded proxy ([Envoy](https://www.envoyproxy.io/))** 
```

Comment:
I kind of wanted to keep the identity of the proxy somewhat hidden.  Users can't configure it in any way, and I'd like the option to switch to something else without anyone noticing.

Reply:
Done - Used the generic term of Proxy